tests/e2e/data/typechecker-simulator-contents.ts

```diff
@@ -0,0 +1,55 @@
+export const oneRowTable = `
+┌────────────────┬────────┐
+│ Contract       │ Result │
+├────────────────┼────────┤
+│ hello-tacos.tz │ Valid  │
+└────────────────┴────────┘
+`.trimStart()
+
+export const oneRowTableSimulateResult = `
+┌────────────────┬────────────────────┐
```

Comment:
Is this how we want the output to look?

Reply:
Yep this is the best solution after discussing with @mweichert